### DISCUSSION TITLE
Jailmaker vs Ubernerd

### DISCUSSION THREAD

**Discussion by ghost** [Jip-Hop/ubernerd#6] [Q&A]
Hello! Thank you for doing this for the community. I have been using jailmaker successfully for a long while now thanks to your help. Is this project (ubernerd) the successor to jailmaker? I am attempting to set up k3s (the built in k3s implementation is... not ideal for my use case) inside of one of these jails so that I can use the local path provisioner + bind mounting from my TN Scale host. Is this something I should be doing with ubernerd or should i stick to jailmaker?

**Comment by Jip-Hop** [comment]
Perhaps it's the future, but if you ask me it's not the present because of the open issues on ubernerd. If they're not dealbreakers to you then by all means go ahead and use it.

Also please see this reply: https://github.com/Jip-Hop/ubernerd/discussions/5#discussioncomment-7227159
